src/main/java/org/littleshoot/proxy/ServerConnectionFailureHttpResponseComposer.java

```diff
@@ -0,0 +1,9 @@
+package org.littleshoot.proxy;
+
+
+import io.netty.handler.codec.http.FullHttpResponse;
+import io.netty.handler.codec.http.HttpRequest;
+
+public interface ServerConnectionFailureHttpResponseComposer {
```

Comment:
unsure about the naming on this. it seems wrong to limit it to just server connection failures. how about just `HttpResponseComposer`?